### ISSUE TITLE
Add asynchronous method for Authorize and capture or increase time that is used with synchronous method

### ISSUE THREAD

**Issue by ChristianAF** [woocommerce/woocommerce-gateway-amazon-pay#181]
"The plugin doesn't have an option to automatically capture funds if an asynchronous method is used. There is no such option in the plugin. How this can be a problem with our wordpress if your plugin doesn't even have this option.

Your plugin must use a synchronous method if we want to capture funds automatically. However, there is a certain time limit that is used with synchronous method and sometimes it takes for Amazon more time to capture funds than the period that you have enabled with synchronous method. This is a major issue because if it takes more time for Amazon to approve the purchase than the time period that you have setup with the synchronous method, the sale doesn't go through. Which means that we lose a sale and customers get annoyed that our website doesn't work properly. As you can imagine this is THE MOST SERIOUS ISSUE THAT A WEBSITE CAN HAVE WITH A PAYMENT PROCESSOR. The only current solution is to use an asynchronous method, but it doesn't allow for automatic fund capture.

So 2 possible solutions (as I see it) would be either add automatic fund capture for the asynchronous method or greatly increase time that is used with the synchronous method."

**WordPress Environment**
<details>
```
### WordPress Environment ###

WordPress address (URL): https://healthyline.com
Site address (URL): https://healthyline.com
WC Version: 5.7.2
REST API Version: ✔ 5.7.2
WC Blocks Version: ✔ 5.7.2
Action Scheduler Version: ✔ 3.2.1
WC Admin Version: ✔ 2.6.5
Log Directory Writable: ✔
WP Version: ❌ 5.8.4 - There is a newer version of WordPress available (5.9.2)
WP Multisite: –
WP Memory Limit: 1 GB
WP Debug Mode: –
WP Cron: ✔
Language: en_US
External object cache: –

### Server Environment ###

Server Info: Apache
PHP Version: 7.4.28
PHP Post Max Size: 256 MB
PHP Time Limit: 300
PHP Max Input Vars: 2000
cURL Version: 7.81.0
OpenSSL/1.1.1n

SUHOSIN Installed: –
MySQL Version: 5.6.51-log
Max Upload Size: 64 MB
Default Timezone is UTC: ✔
fsockopen/cURL: ✔
SoapClient: ✔
DOMDocument: ✔
GZip: ✔
Multibyte String: ✔
Remote Post: ✔
Remote Get: ✔

### Database ###

WC Database Version: 5.7.2
WC Database Prefix: msbxl_
Total Database Size: 2960.87MB
Database Data Size: 2023.73MB
Database Index Size: 937.14MB
msbxl_woocommerce_sessions: Data: 120.66MB + Index: 5.97MB + Engine MyISAM
msbxl_woocommerce_api_keys: Data: 0.06MB + Index: 0.07MB + Engine MyISAM
msbxl_woocommerce_attribute_taxonomies: Data: 0.00MB + Index: 0.01MB + Engine MyISAM
msbxl_woocommerce_downloadable_product_permissions: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
msbxl_woocommerce_order_items: Data: 4.24MB + Index: 1.97MB + Engine MyISAM
msbxl_woocommerce_order_itemmeta: Data: 50.48MB + Index: 15.27MB + Engine MyISAM
msbxl_woocommerce_tax_rates: Data: 0.00MB + Index: 0.01MB + Engine MyISAM
msbxl_woocommerce_tax_rate_locations: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
msbxl_woocommerce_shipping_zones: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
msbxl_woocommerce_shipping_zone_locations: Data: 0.00MB + Index: 0.01MB + Engine MyISAM
msbxl_woocommerce_shipping_zone_methods: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
msbxl_woocommerce_payment_tokens: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
msbxl_woocommerce_payment_tokenmeta: Data: 0.00MB + Index: 0.01MB + Engine MyISAM
msbxl_woocommerce_log: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
job: Data: 2.14MB + Index: 0.00MB + Engine MyISAM
msbxl_abandoned_cart_aelia_currency: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
msbxl_actionscheduler_actions: Data: 34.82MB + Index: 13.38MB + Engine MyISAM
msbxl_actionscheduler_claims: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
msbxl_actionscheduler_groups: Data: 0.00MB + Index: 0.01MB + Engine MyISAM
msbxl_actionscheduler_logs: Data: 24.46MB + Index: 16.30MB + Engine MyISAM
msbxl_ac_abandoned_cart_history: Data: 1.28MB + Index: 0.05MB + Engine MyISAM
msbxl_ac_email_templates: Data: 0.03MB + Index: 0.00MB + Engine MyISAM
msbxl_ac_guest_abandoned_cart_history: Data: 0.01MB + Index: 0.01MB + Engine MyISAM
msbxl_ac_link_clicked_email: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
msbxl_ac_notifications: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
msbxl_ac_notifications_meta: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
msbxl_ac_opened_emails: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
msbxl_ac_sent_history: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
msbxl_ac_tiny_urls: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
msbxl_advanced_notifications: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
msbxl_advanced_notification_triggers: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
msbxl_aepc_custom_audiences: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
msbxl_aepc_logs: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
msbxl_aiowps_events: Data: 0.76MB + Index: 0.07MB + Engine MyISAM
msbxl_aiowps_failed_logins: Data: 0.58MB + Index: 0.16MB + Engine MyISAM
msbxl_aiowps_global_meta: Data: 3.63MB + Index: 0.00MB + Engine MyISAM
msbxl_aiowps_login_activity: Data: 0.47MB + Index: 0.11MB + Engine MyISAM
msbxl_aiowps_login_lockdown: Data: 0.07MB + Index: 0.02MB + Engine MyISAM
msbxl_aiowps_permanent_block: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
msbxl_automatewoo_abandoned_carts: Data: 0.12MB + Index: 0.02MB + Engine MyISAM
msbxl_automatewoo_customers: Data: 0.94MB + Index: 2.01MB + Engine MyISAM
msbxl_automatewoo_customer_meta: Data: 0.00MB + Index: 0.01MB + Engine MyISAM
msbxl_automatewoo_events: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
msbxl_automatewoo_guests: Data: 0.56MB + Index: 1.06MB + Engine MyISAM
msbxl_automatewoo_guest_meta: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
msbxl_automatewoo_logs: Data: 0.04MB + Index: 0.09MB + Engine MyISAM
msbxl_automatewoo_log_meta: Data: 0.30MB + Index: 0.19MB + Engine MyISAM
msbxl_automatewoo_queue: Data: 0.00MB + Index: 0.02MB + Engine MyISAM
msbxl_automatewoo_queue_meta: Data: 0.02MB + Index: 0.03MB + Engine MyISAM
msbxl_blc_filters: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
msbxl_blc_instances: Data: 1.52MB + Index: 0.25MB + Engine InnoDB
msbxl_blc_links: Data: 2.52MB + Index: 0.34MB + Engine InnoDB
msbxl_blc_synch: Data: 0.06MB + Index: 0.02MB + Engine InnoDB
msbxl_clearsale_usa: Data: 0.31MB + Index: 0.07MB + Engine MyISAM
msbxl_commentmeta: Data: 17.50MB + Index: 4.83MB + Engine InnoDB
msbxl_comments: Data: 40.58MB + Index: 26.09MB + Engine InnoDB
msbxl_ep_template_library_cat: Data: 0.01MB + Index: 0.01MB + Engine MyISAM
msbxl_ep_template_library_cat_post: Data: 0.06MB + Index: 0.04MB + Engine MyISAM
msbxl_ep_template_library_post: Data: 0.49MB + Index: 0.06MB + Engine MyISAM
msbxl_et_social_stats: Data: 0.01MB + Index: 0.00MB + Engine MyISAM
msbxl_ewwwio_images: Data: 1.52MB + Index: 2.73MB + Engine InnoDB
msbxl_e_submissions: Data: 0.01MB + Index: 0.04MB + Engine MyISAM
msbxl_e_submissions_actions_log: Data: 0.00MB + Index: 0.01MB + Engine MyISAM
msbxl_e_submissions_values: Data: 0.00MB + Index: 0.01MB + Engine MyISAM
msbxl_feedmanager_channel: Data: 0.00MB + Index: 0.01MB + Engine MyISAM
msbxl_feedmanager_country: Data: 0.01MB + Index: 0.02MB + Engine MyISAM
msbxl_feedmanager_errors: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
msbxl_feedmanager_feed_status: Data: 0.00MB + Index: 0.01MB + Engine MyISAM
msbxl_feedmanager_field_categories: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
msbxl_feedmanager_product_feed: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
msbxl_feedmanager_product_feedmeta: Data: 0.01MB + Index: 0.01MB + Engine MyISAM
msbxl_feedmanager_source: Data: 0.00MB + Index: 0.01MB + Engine MyISAM
msbxl_geodir_attachments: Data: 0.01MB + Index: 0.00MB + Engine MyISAM
msbxl_geodir_countries: Data: 0.03MB + Index: 0.00MB + Engine MyISAM
msbxl_geodir_custom_fields: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
msbxl_geodir_custom_sort_fields: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
msbxl_geodir_gd_place_detail: Data: 0.01MB + Index: 0.00MB + Engine MyISAM
msbxl_geodir_location_seo: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
msbxl_geodir_post_icon: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
msbxl_geodir_post_locations: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
msbxl_geodir_post_neighbourhood: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
msbxl_geodir_post_review: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
msbxl_geodir_term_meta: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
msbxl_ia_savedcarts: Data: 0.39MB + Index: 0.03MB + Engine MyISAM
msbxl_iw_payplans: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
msbxl_links: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
msbxl_lws_wr_achieved_log: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
msbxl_lws_wr_historic: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
msbxl_options: Data: 23.25MB + Index: 4.22MB + Engine InnoDB
msbxl_pmxe_exports: Data: 0.94MB + Index: 0.00MB + Engine MyISAM
msbxl_pmxe_google_cats: Data: 0.26MB + Index: 0.05MB + Engine MyISAM
msbxl_pmxe_posts: Data: 2.41MB + Index: 1.05MB + Engine MyISAM
msbxl_pmxe_templates: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
msbxl_postmeta: Data: 653.00MB + Index: 319.66MB + Engine InnoDB
msbxl_posts: Data: 631.00MB + Index: 387.06MB + Engine InnoDB
msbxl_ppress_forms: Data: 0.00MB + Index: 0.01MB + Engine MyISAM
msbxl_ppress_formsmeta: Data: 0.03MB + Index: 0.02MB + Engine MyISAM
msbxl_ppress_meta_data: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
msbxl_ratings: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
msbxl_redirection_404: Data: 0.60MB + Index: 0.11MB + Engine MyISAM
msbxl_redirection_groups: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
msbxl_redirection_items: Data: 0.00MB + Index: 0.02MB + Engine MyISAM
msbxl_redirection_logs: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
msbxl_smush_dir_images: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
msbxl_tcb_api_error_log: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
msbxl_termmeta: Data: 0.04MB + Index: 0.04MB + Engine MyISAM
msbxl_terms: Data: 0.06MB + Index: 0.03MB + Engine InnoDB
msbxl_term_relationships: Data: 91.67MB + Index: 54.63MB + Engine InnoDB
msbxl_term_taxonomy: Data: 0.06MB + Index: 0.03MB + Engine InnoDB
msbxl_tm_taskmeta: Data: 0.00MB + Index: 0.01MB + Engine MyISAM
msbxl_tm_tasks: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
msbxl_tve_leads_contacts: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
msbxl_tve_leads_contact_download: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
msbxl_tve_leads_event_log: Data: 0.29MB + Index: 0.22MB + Engine MyISAM
msbxl_tve_leads_form_variations: Data: 0.01MB + Index: 0.00MB + Engine MyISAM
msbxl_tve_leads_group_options: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
msbxl_tve_leads_saved_group_options: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
msbxl_tve_leads_split_test: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
msbxl_tve_leads_split_test_items: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
msbxl_usermeta: Data: 42.55MB + Index: 18.03MB + Engine InnoDB
msbxl_users: Data: 0.48MB + Index: 0.42MB + Engine InnoDB
msbxl_warranty_table: Data: 0.23MB + Index: 0.01MB + Engine MyISAM
msbxl_warranty_table_notes: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
msbxl_wcpdf_invoice_number: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
msbxl_wc_admin_notes: Data: 0.03MB + Index: 0.00MB + Engine MyISAM
msbxl_wc_admin_note_actions: Data: 0.01MB + Index: 0.01MB + Engine MyISAM
msbxl_wc_category_lookup: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
msbxl_wc_customer_lookup: Data: 0.60MB + Index: 0.52MB + Engine MyISAM
msbxl_wc_download_log: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
msbxl_wc_order_coupon_lookup: Data: 0.15MB + Index: 0.27MB + Engine MyISAM
msbxl_wc_order_product_lookup: Data: 1.26MB + Index: 1.18MB + Engine MyISAM
msbxl_wc_order_stats: Data: 0.73MB + Index: 0.49MB + Engine MyISAM
msbxl_wc_order_tax_lookup: Data: 0.01MB + Index: 0.01MB + Engine MyISAM
msbxl_wc_product_meta_lookup: Data: 0.05MB + Index: 0.07MB + Engine MyISAM
msbxl_wc_reserved_stock: Data: 0.01MB + Index: 0.01MB + Engine MyISAM
msbxl_wc_tax_rate_classes: Data: 0.00MB + Index: 0.01MB + Engine MyISAM
msbxl_wc_warranty_products: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
msbxl_wc_webhooks: Data: 0.01MB + Index: 0.00MB + Engine MyISAM
msbxl_wc_zapier_history: Data: 19.05MB + Index: 1.53MB + Engine MyISAM
msbxl_wfBlockedIPLog: Data: 0.05MB + Index: 0.03MB + Engine MyISAM
msbxl_wfBlocks7: Data: 0.03MB + Index: 0.02MB + Engine MyISAM
msbxl_wfConfig: Data: 4.46MB + Index: 0.01MB + Engine MyISAM
msbxl_wfCrawlers: Data: 0.01MB + Index: 0.01MB + Engine MyISAM
msbxl_wfFileChanges: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
msbxl_wfFileMods: Data: 9.38MB + Index: 1.67MB + Engine MyISAM
msbxl_wfHits: Data: 8.25MB + Index: 0.76MB + Engine MyISAM
msbxl_wfHoover: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
msbxl_wfIssues: Data: 0.18MB + Index: 0.02MB + Engine MyISAM
msbxl_wfKnownFileList: Data: 5.43MB + Index: 0.52MB + Engine MyISAM
msbxl_wfLiveTrafficHuman: Data: 0.11MB + Index: 0.17MB + Engine MyISAM
msbxl_wfLocs: Data: 0.01MB + Index: 0.01MB + Engine MyISAM
msbxl_wfLogins: Data: 0.44MB + Index: 0.10MB + Engine MyISAM
msbxl_wfls_2fa_secrets: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
msbxl_wfls_settings: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
msbxl_wfNotifications: Data: 0.01MB + Index: 0.00MB + Engine MyISAM
msbxl_wfPendingIssues: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
msbxl_wfReverseCache: Data: 0.01MB + Index: 0.00MB + Engine MyISAM
msbxl_wfSNIPCache: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
msbxl_wfStatus: Data: 0.23MB + Index: 0.09MB + Engine MyISAM
msbxl_wfTrafficRates: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
msbxl_woocommerce_amazon_buyer_index: Data: 0.01MB + Index: 0.01MB + Engine MyISAM
msbxl_woocommerce_bundled_itemmeta: Data: 0.00MB + Index: 0.01MB + Engine MyISAM
msbxl_woocommerce_bundled_items: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
msbxl_woocommerce_exported_csv_items: Data: 1.64MB + Index: 0.02MB + Engine MyISAM
msbxl_woocommerce_shipping_table_rates: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
msbxl_woocommerce_square_customers: Data: 0.01MB + Index: 0.02MB + Engine MyISAM
msbxl_wpam_actions: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
msbxl_wpam_affiliates: Data: 0.52MB + Index: 0.02MB + Engine MyISAM
msbxl_wpam_affiliates_fields: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
msbxl_wpam_creatives: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
msbxl_wpam_events: Data: 0.08MB + Index: 0.04MB + Engine MyISAM
msbxl_wpam_impressions: Data: 0.01MB + Index: 0.00MB + Engine MyISAM
msbxl_wpam_messages: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
msbxl_wpam_paypal_logs: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
msbxl_wpam_tracking_tokens: Data: 190.27MB + Index: 36.33MB + Engine MyISAM
msbxl_wpam_tracking_tokens_purchase_logs: Data: 0.05MB + Index: 0.04MB + Engine MyISAM
msbxl_wpam_transactions: Data: 0.86MB + Index: 0.08MB + Engine MyISAM
msbxl_wpbdp_fees: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
msbxl_wpbdp_form_fields: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
msbxl_wpbdp_listing_fees: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
msbxl_wpbdp_payments: Data: 0.00MB + Index: 0.01MB + Engine MyISAM
msbxl_wpbdp_payments_items: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
msbxl_wpbdp_submit_state: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
msbxl_wpforms_entries: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
msbxl_wpforms_entry_fields: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
msbxl_wpforms_entry_meta: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
msbxl_wpforms_tasks_meta: Data: 0.01MB + Index: 0.01MB + Engine MyISAM
msbxl_wpsm_tables: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
msbxl_wp_mail_debug_emails: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
msbxl_wsal_metadata: Data: 3.32MB + Index: 2.75MB + Engine MyISAM
msbxl_wsal_occurrences: Data: 0.12MB + Index: 0.17MB + Engine MyISAM
msbxl_wsal_options: Data: 0.01MB + Index: 0.00MB + Engine MyISAM
msbxl_wt_iew_action_history: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
msbxl_wt_iew_cron: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
msbxl_wt_iew_ftp: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
msbxl_wt_iew_mapping_template: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
msbxl_yith_wcwl: Data: 2.39MB + Index: 1.94MB + Engine MyISAM
msbxl_yith_wcwl_lists: Data: 3.48MB + Index: 4.55MB + Engine MyISAM
msbxl_yoast_indexable: Data: 8.42MB + Index: 3.19MB + Engine MyISAM
msbxl_yoast_indexable_hierarchy: Data: 0.31MB + Index: 1.00MB + Engine MyISAM
msbxl_yoast_migrations: Data: 0.00MB + Index: 0.01MB + Engine MyISAM
msbxl_yoast_primary_term: Data: 0.03MB + Index: 0.04MB + Engine MyISAM
msbxl_yoast_seo_links: Data: 2.28MB + Index: 0.83MB + Engine MyISAM
msbxl_yoast_seo_meta: Data: 1.72MB + Index: 1.45MB + Engine MyISAM

### Post Type Counts ###

acf: 37
acf-field: 1821
acf-field-group: 89
advanced_ads: 3
aiosrs-schema: 8
amn_mi-lite: 2
amn_om: 1
attachment: 6131
aw_workflow: 9
blocks: 1
business: 109
custom_css: 1
customize_changeset: 1
cwginstocknotifier: 1067
elementor_library: 21
flamingo_contact: 7059
flamingo_inbound: 14281
gd_place: 14
gift_card: 24
healthy-stone: 12
illness-testimonial: 18
klaviyo_shop_cart: 2261536
landingpage_f: 1
leadpages_post: 1
mc4wp-form: 1
nav_menu_item: 179
oembed_cache: 1738
omapi: 70
page: 421
post: 193
postman_sent_mail: 254
product: 239
product_variation: 326
pw_bogo: 2
revision: 2482
rp_shop_giftcard: 1
seedprod: 1
shop_bogof_rule: 6
shop_coupon: 56798
shop_order: 22033
shop_order_refund: 1760
shop_subscription: 7
tve_form_type: 10
tve_lead_group: 2
useful-videos: 10
wafs: 1
wpbdp_listing: 1
wpcf7_contact_form: 10
wpforms: 1
yith_wc_surveys: 4

### Security ###

Secure connection (HTTPS): ✔
Hide errors from visitors: ✔

### Active Plugins (104) ###

Advanced Custom Fields: Date and Time Picker: by Per Soderlind – 2.1.5
Advanced Custom Fields: Options Page: by Elliot Condon – 2.1.0
Advanced Custom Fields: Repeater Field: by Elliot Condon – 2.1.0
ACF RGBA Color Picker: by Thomas Meyer – 1.2.2
ACF to REST API: by Aires Gonçalves – 3.3.2
Adminimize: by Frank Bültge – 1.11.7
Advanced Cron Manager: by BracketSpace – 2.4.1
Advanced Custom Fields: by Delicious Brains – 5.10.2
Affiliates Manager WooCommerce Coupons Integration: by wp.insider
affmngr – 1.0.2

WP Affiliate Manager: by wp.insider
wpaffiliatemgr – 9

Ajax Search Lite: by Ernest Marcinko – 4.9.4
Akismet Anti-Spam: by Automattic – 4.2
All In One WP Security: by Tips and Tricks HQ
Peter Petreski
Ruhul
Ivy – 4.4.9

Angell EYE Updater: by Angell EYE – 2.0.17
Application Passwords: by George Stephanis – 0.1.3
AutomateWoo: by WooCommerce – 5.5.5
Back In Stock Notifier for WooCommerce | WooCommerce Waitlist Pro: by codewoogeek – 2.3.3.1
Element Pack Pro: by BdThemes – 5.11.1
Broken Link Checker: by WPMU DEV – 1.11.16
Classic Editor: by WordPress Contributors – 1.6.2
ClearSale: by ClearSale – 1.2.0
Comment Images Reloaded: by WP Puzzle – 2.2.1
Honeypot for Contact Form 7: by Nocean – 2.1
Contact Form 7 Captcha: by 247wd – 0.1.0
Contact Form 7: by Takayuki Miyoshi – 5.4.2
Conversion Tracking for Bing in WooCommerce: by Worcester Web Studio – 1.0.7.2
Csv codes to woocommerce coupons: by Web Design Sun – 0.2
Duplicate Page: by mndpsingh287 – 4.4.4
Google Tag Manager for Wordpress: by Thomas Geiger – 1.13.1
Google Customer Reviews for WooCommerce: by eCreations – 2.6.12
Elementor Pro: by Elementor.com – 3.4.1
Elementor: by Elementor.com – 3.4.4
Enable Gzip Compression: by Moki-Moki Ios – 1.0.3
Export WooCommerce customer list PRO: by Pi Websolution – 1.1.3
Facebook for WooCommerce: by Facebook – 2.6.5
Flamingo: by Takayuki Miyoshi – 2.2.2
GA Google Analytics: by Jeff Starr – 20210719
Head, Footer and Post Injections: by Stefano Lissa – 3.2.2
WooCommerce: No PO Boxes: by Maje Media LLC – 2.1.2
NitroPack: by NitroPack LLC – 1.5.7
OptinMonster: by OptinMonster Team – 2.6.5
Order / Coupon / Subscription Export Import Plugin for WooCommerce (BASIC): by WebToffee – 2.1.1
Password Strength for WooCommerce: by Aspen Grove Studios – 1.0.2
PayPal for WooCommerce: by Angell EYE – 2.5.17
Virtual Robots.txt: by Marios Alexandrou – 1.10
Post SMTP: by Yehuda Hassine – 2.0.23
Product Recommendation Quiz for WooCommerce: by RevenueHunt – 2.0.9
PW WooCommerce BOGO Pro: by Pimwick
LLC – 2.151

Fixed Widget: by Thomas Maier
Max Bond – 5.3.0

Really Simple CAPTCHA: by Takayuki Miyoshi – 2.1
Redirection: by John Godley – 5.2.3
Retainful - Abandoned Cart Recovery Emails and Next Order Coupons: by Retainful – 2.6.4
Sassy Social Share: by Team Heateor – 3.3.24
Sezzle WooCommerce Payment: by Sezzle – 3.1.15
Simple 301 Redirects: by WPDeveloper – 2.0.6
SVG Support: by Benbodhi – 2.3.19
Table Maker: by Wpsoul – 1.9.1
TablePress: by Tobias Bäthge – 1.14
Advanced Editor Tools (previously TinyMCE Advanced): by Automattic – 5.6.0
Ultimate Category Excluder: by Marios Alexandrou – 1.5
User Role Editor: by Vladimir Garagulya – 4.60.2
WDS Clyde: by Web Design Sun Team – 1.0.1
WDS Warranty: by Web Design Sun Team – 1.1.0
WebP Converter for Media: by Mateusz Gbiorczyk – 3.2.2
Product Gallery Slider for WooCommerce: by codeixer – 2.2
WooCommerce Account Funds: by Themesquad – 2.5.3 (update to version 2.6.4 is available)
WooCommerce Advanced Notifications: by WooCommerce – 1.2.32 (update to version 1.2.36 is available)
WooCommerce Availability Notifications: by ThemePlugger – 1.0.3
WooCommerce Buy One Get One Free: by Oscar Gare – 2.2.4 (update to version 3.1.0 is available)
WooCommerce Compare List: by Themeisle – 1.1.2
WooCommerce Customer/Order/Coupon Export: by SkyVerge – 5.3.3
WooCommerce Export Customer Email: by MH Mithu – 1.3
WooCommerce Amazon Pay: by WooCommerce – 2.1.1 (update to version 2.1.2 is available)
WooCommerce Pixel Manager: by woopt – 1.11.6
WooCommerce Google Analytics Pro: by SkyVerge – 1.11.0 (update to version 1.11.1 is available)
WooCommerce Google Ads Dynamic Remarketing: by woopt – 1.8.1
WooCommerce Min/Max Quantities: by WooCommerce – 2.4.24
OrderCup: by OrderCup – 3.1
WooCommerce PDF Invoices & Packing Slips: by WP Overnight – 2.13.1
WooCommerce Sequential Order Numbers: by SkyVerge – 1.9.5
WooCommerce Shipment Tracking: by WooCommerce – 1.6.29 (update to version 1.6.31 is available)
WooCommerce Square: by WooCommerce – 2.6.0 (update to version 2.9.1 is available)
WooCommerce - Store Exporter Deluxe: by Visser Labs – 4.2
WooCommerce Table Rate Shipping: by WooCommerce – 3.0.34 (update to version 3.0.36 is available)
WooCommerce Zapier: by OM4 Software – 2.2.2 (update to version 2.3.0 is available)
WooCommerce: by Automattic – 5.7.2 (update to version 6.3.1 is available)
MyRewards: by Long Watch Studio – 4.8.1
Wordfence Security: by Wordfence – 7.5.8
WordPress Importer: by wordpressdotorg – 0.7
Yoast SEO: by Team Yoast – 17.2.1
WP Activity Log for WooCommerce: by WP White Security – 1.3.1
WP All Export Pro: by Soflyy – 1.7.1
WP Crontrol: by John Blackbourn & crontributors – 1.10.0
WP Force SSL: by WebFactory Ltd – 1.57
WP-PostRatings: by Lester 'GaMerZ' Chan – 1.89
WP Product Feed Manager: by Michel Jongbloed – 1.34.0
Schema Pro: by Brainstorm Force – 2.3.0
WP Activity Log: by WP White Security – 4.3.4
ProfilePress: by ProfilePress Team – 3.1.19
WPForms Geolocation: by WPForms – 2.0.0
WPForms: by WPForms – 1.6.9
YITH WooCommerce Ajax Search: by YITH – 1.10.1
YITH WooCommerce Gift Cards Premium: by YITH – 3.5.1
YITH WooCommerce Wishlist: by YITH – 3.1.1

### Inactive Plugins (31) ###

Abandoned Cart Pro for WooCommerce: by Tyche Softwares – 8.4.0
Action Scheduler: by Automattic – 3.3.0
AddToAny Share Buttons: by AddToAny – 1.7.46
Advanced Ads: by Thomas Maier
Advanced Ads GmbH – 1.28.0

Autoptimize: by Frank Goossens (futtta) – 2.9.2
Better Search Replace: by Delicious Brains – 1.3.4
Coming Soon Page, Maintenance Mode & Landing Pages by SeedProd: by SeedProd – 6.6.0
Csv codes to woocommerce coupons: by Web Design Sun – 0.1
InfusedWoo Pro: by Mark Joseph – 3.15.4
Klarna Checkout for WooCommerce: by Krokedil – 2.6.1
Klarna Order Management for WooCommerce: by klarna
krokedil – 1.6.7

Product CSV Import Export (BASIC): by WebToffee – 2.0.9
Query Monitor: by John Blackbourn – 3.7.1
Regenerate Thumbnails: by Alex Mills (Viper007Bond) – 3.1.5
REST API Log: by Pete Nelson – 1.6.8
Tagove - Live Chat Software: by Tagove Team – 1.5
UsageDD: by Dion Designs – 1.4.7
W3 Total Cache: by BoldGrid – 2.1.8
WooCommerce - Store Exporter: by Visser Labs – 2.7
WooCommerce Conversion Tracking Pro: by weDevs – 1.0.6
WooCommerce Infusionsoft: by Objects – 1.0.6
WooCommerce Name Your Price: by Kathy Darling – 3.1.2 (update to version 3.3.9 is available)
WooCommerce Product Stock Alert: by WC Marketplace – 1.7.3
WooCommerce Shipping & Tax: by WooCommerce – 1.25.18 (update to version 1.25.24 is available)
WP-Optimize - Clean, Compress, Cache: by David Anderson
Ruhani Rabin
Team Updraft – 3.1.12

WP All Export: by Soflyy – 1.3.1
WP Disable: by optimisation.io - jody nesbitt – 1.6.1
WP Fastest Cache: by Emre Vona – 0.9.4
WP Fastest Cache Premium: by Emre Vona – 1.5.9
WPForms Lite: by WPForms – 1.6.9
WP reCaptcha Integration: by weDevs – 1.2.4

### Dropin Plugins (2) ###

advanced-cache.php: advanced-cache.php
maintenance.php: maintenance.php

### Settings ###

API Enabled: ✔
Force SSL: ✔
Currency: USD ($)
Currency Position: left
Thousand Separator: ,
Decimal Separator: .
Number of Decimals: 2
Taxonomies: Product Types: bundle (bundle)
clyde_contract (clyde_contract)
covered (covered)
external (external)
gift-card (gift-card)
grouped (grouped)
simple (simple)
subscription (subscription)
variable (variable)
variable subscription (variable-subscription)

Taxonomies: Product Visibility: exclude-from-catalog (exclude-from-catalog)
exclude-from-search (exclude-from-search)
featured (featured)
outofstock (outofstock)
rated-1 (rated-1)
rated-2 (rated-2)
rated-3 (rated-3)
rated-4 (rated-4)
rated-5 (rated-5)

Connected to WooCommerce.com: ✔

### WC Pages ###

Shop base: #198 - /shop/
Cart: #77 - /cart/
Checkout: #79 - /checkout/
My account: #199 - /account/
Terms and conditions: #203 - /terms-conditions/

### Theme ###

Name: Web Design Sun® Theme
Version: 1.0
Author URL: https://www.webdesignsun.com
Child Theme: ❌ – If you are modifying WooCommerce on a parent theme that you did not build personally we recommend using a child theme. See: How to create a child theme
WooCommerce Support: ✔

### Templates ###

Overrides: webdesingsun/woocommerce/archive-product.php
webdesingsun/woocommerce/cart/cart-empty.php
webdesingsun/woocommerce/cart/cart-item-data.php
webdesingsun/woocommerce/cart/cart-shipping.php
webdesingsun/woocommerce/cart/cart-totals.php
webdesingsun/woocommerce/cart/cart.php
webdesingsun/woocommerce/cart/cross-sells.php
webdesingsun/woocommerce/cart/mini-cart.php
webdesingsun/woocommerce/cart/proceed-to-checkout-button.php
webdesingsun/woocommerce/cart/shipping-calculator.php
webdesingsun/woocommerce/checkout/form-checkout.php
webdesingsun/woocommerce/checkout/form-coupon.php
webdesingsun/woocommerce/checkout/form-shipping.php
webdesingsun/woocommerce/checkout/payment-method.php
webdesingsun/woocommerce/checkout/payment.php
webdesingsun/woocommerce/checkout/thankyou.php
webdesingsun/woocommerce/content-product.php
webdesingsun/woocommerce/content-single-product.php
webdesingsun/woocommerce/global/breadcrumb.php
webdesingsun/woocommerce/global/form-login.php
webdesingsun/woocommerce/global/quantity-input.php
webdesingsun/woocommerce/global/sidebar.php
webdesingsun/woocommerce/global/wrapper-end.php
webdesingsun/woocommerce/global/wrapper-start.php
webdesingsun/woocommerce/loop/add-to-cart.php
webdesingsun/woocommerce/loop/loop-end.php
webdesingsun/woocommerce/loop/loop-start.php
webdesingsun/woocommerce/loop/no-products-found.php
webdesingsun/woocommerce/loop/orderby.php
webdesingsun/woocommerce/loop/pagination.php
webdesingsun/woocommerce/loop/price.php
webdesingsun/woocommerce/loop/rating.php
webdesingsun/woocommerce/loop/result-count.php
webdesingsun/woocommerce/loop/sale-flash.php
webdesingsun/woocommerce/myaccount/downloads.php
webdesingsun/woocommerce/myaccount/form-login.php
webdesingsun/woocommerce/myaccount/navigation.php
webdesingsun/woocommerce/myaccount/view-order.php
webdesingsun/woocommerce/single-product/add-to-cart/simple.php
webdesingsun/woocommerce/single-product/price.php
webdesingsun/woocommerce/single-product/product-attributes.php
webdesingsun/woocommerce/single-product/rating.php
webdesingsun/woocommerce/single-product/related.php
webdesingsun/woocommerce/single-product/review-meta.php
webdesingsun/woocommerce/single-product/title.php
webdesingsun/woocommerce/single-product/up-sells.php
webdesingsun/woocommerce/single-product-reviews.php
webdesingsun/woocommerce/single-product.php


### Square ###

Environment: Production
Tokenization Enabled: –
Debug Mode: Off

### WooCommerce Customer/Order/Coupon Export ###

Automated exports: 2
Custom formats: 3
Next export: Not scheduled

### Action Scheduler ###

Complete: 79,892
Oldest: 2022-03-11 14:13:32 +0000
Newest: 2022-03-18 23:10:06 +0000

Failed: 411
Oldest: 2018-06-17 11:59:21 +0000
Newest: 2022-03-18 21:08:20 +0000

Pending: 35
Oldest: 2022-03-18 23:17:51 +0000
Newest: 2022-03-24 21:13:09 +0000


### Templates ###

Extended logging mode for Store Exporter Deluxe: ✔Inactive Turn on Logging mode
Debugging mode for Store Exporter Deluxe: ✔Inactive Turn on Debugging mode
Scheduled Export to FTP: ✔
Scheduled Export to SFTP: ❌ The required SFTP functions are not available; ssh2_connect()
ssh2_auth_password()
ssh2_sftp(). Contact your hosting provider.


### WooCommerce Zapier ###

Database Version: ✔ 13
Detailed Logging Enabled: No
Legacy Mode Disabled: ✔ Yes
SSL: ✔ Yes
REST API Authentication Key(s): ❌ 0
Task History Record Count: ✔ 60809
Webhook #271: Zapier #subscription:8550283
- Status: active
- Trigger: order.status_changed
- Delivery Count (Last 30 Days): 1694
Webhook #267: Zapier #subscription:8549844
- Status: active
- Trigger: order.status_changed
- Delivery Count (Last 30 Days): 1693
Webhook #264: Zapier #subscription:7552574
- Status: active
- Trigger: order.status_changed
- Delivery Count (Last 30 Days): 1693
Webhook #263: Zapier #subscription:7551326
- Status: active
- Trigger: order.status_changed
- Delivery Count (Last 30 Days): 1693
Webhook #262: Zapier #subscription:7551300
- Status: disabled
- Trigger: order.status_changed
- Delivery Count (Last 30 Days): 0
Webhook #260: Zapier #subscription:7551020
- Status: active
- Trigger: order.status_changed
- Delivery Count (Last 30 Days): 1693
Webhook #258: Zapier #subscription:7550127
- Status: active
- Trigger: order.status_changed
- Delivery Count (Last 30 Days): 1694
Webhook #253: Zapier #subscription:7497993
- Status: active
- Trigger: order.status_changed
- Delivery Count (Last 30 Days): 1694
Webhook #252: Zapier #subscription:7469946
- Status: active
- Trigger: order.status_changed
- Delivery Count (Last 30 Days): 1694
Webhook #251: Zapier #subscription:7469766
- Status: active
- Trigger: order.status_changed
- Delivery Count (Last 30 Days): 1693
Deleted Webhook Delivery Count: 45568

### Status report information ###

Generated at: 2022-03-18 18:12:09 -05:00
```
</details>

**Comment by dpanta94** [comment]
Hey @ChrissiePollock,

The main reason that implementing a method to automatically capture funds of orders created through async authorization has been given a low priority is that basically we can't know when every merchant would like the funds to be captured.

There are many merchants that have created custom integrations with shipping plugins. For example, when an order is ready to be shipped that's where they have hooked with custom code in order to capture the funds as well.

Without taking into consideration external plugins, we could provide 2 places where an async's order could have its funds captured automatically.

When changing an order's status from on-hold to processing or completed. Which again would require manual work or custom code (to have the order's status automatically changed), as a result not actually solving the issue at hand.
Creating a scheduled action to be executed after a defined amount of time, which would capture the funds of an async created order.

Both of these options could break existing custom integrations of merchants. So none of them can be enforced, the functionality should be configurable through the plugin's settings.

We are going to re-evaluate this feature soon. It could be that i am missing some other potentially better option for implementing that right now.

Thank you!
